dotCMS/build.gradle

```diff
@@ -29,25 +29,44 @@ ant.lifecycleLogLevel = "INFO"
 
 ext {
     // Open the Git repository in the current directory.
-    git = org.ajoberstar.grgit.Grgit.open(file('..'))
 
-    dotcmsReleaseName='dotCMS Platform'
     dotcmsReleaseVersion='3.7.0'
+    dotcmsReleaseBuild = ''
+    dotcmsReleaseName='dotCMS Platform'
     dotcmsReleaseCodename='Panther'
     dotcmsReleaseBuild='999999'
```

Comment:
Do we prefer **dotcmsReleaseBuild='999999'** instead of:
**dotcmsReleaseBuild = git.head().abbreviatedId // abbreviatedId of head() method.**